packages/web/src/common/store/cache/collections/commonSagas.ts

```diff
@@ -591,31 +642,35 @@ function* confirmDeleteAlbum(playlistId, trackIds, userId) {
           )
         ])
 
-        const { blockHash, blockNumber, error } = yield call(
+        const { blockHash, blockNumber, error } = (yield* call(
           audiusBackendInstance.deleteAlbum,
           playlistId,
           trackIds
-        )
+        )) as TransactionReceipt & { error?: Error }
         if (error) throw error
 
-        const confirmed = yield call(confirmTransaction, blockHash, blockNumber)
+        const confirmed = yield* call(
+          confirmTransaction,
+          blockHash,
+          blockNumber
+        )
         if (!confirmed) {
           throw new Error(`Could not confirm delete album for id ${playlistId}`)
         }
         return playlistId
       },
       function* () {
         console.debug(`Successfully deleted album ${playlistId}`)
-        yield put(cacheActions.remove(Kind.COLLECTIONS, [playlistId]))
+        yield* put(cacheActions.remove(Kind.COLLECTIONS, [playlistId]))
       },
       function* ({ error, timeout, message }) {
         console.error(`Failed to delete album ${playlistId}`)
         // Need to revert the deletes now
-        const [playlist, user] = yield all([
+        const [playlist, user] = yield* all([
           select(getCollection, { id: playlistId }),
           select(getAccountUser)
-        ])
-        yield all([
+        ]) as unknown as [Collection, UserCollectionMetadata]
```

Comment:
technically these could both be undefined

we need the cast because `yield* all` just unions all the return types together so it doesn't know the first one is playlist and the second is user